src/app/app.module.ts

```diff
@@ -9,6 +9,7 @@ import { SigsComponent } from "./pages/sigs/sigs.component";
 import { ResourcesComponent } from "./pages/resources/resources.component";
 import { ServiceWorkerModule } from "@angular/service-worker";
 import { environment } from "../environments/environment";
+import { EventCardComponent } from "./event-card/event-card.component";
```

Comment:
This should be placed under the components folder or be a single function in the Events page since it is not a reusable component